packages/salesforcedx-apex-debugger/src/adapter/apexDebug.ts

```diff
@@ -272,14 +346,21 @@ export class ApexDebug extends DebugSession {
               lines: info.lines
             };
             lineNumberMapping.get(info.uri)!.push(validLines);
+            typerefMapping.set(info.typeref, info.uri);
           }
-          this.myBreakpointService.setValidLines(lineNumberMapping);
+          this.myBreakpointService.setValidLines(
+            lineNumberMapping,
+            typerefMapping
+          );
         }
         this.sendResponse({
           request_seq: 1,
```

Comment:
What do the numbers request_seq (1) and seq (0) denote?
Is there a way to pull these magic numbers out into constants so it's clearer what is going on?